station/models.py

```diff
@@ -199,6 +179,18 @@ class Station(models.Model):
     the other models in this app, along with some geographical data.
     """
 
+    owner = models.ForeignKey(User, on_delete=models.CASCADE, null=True, blank=True)
+
+    PERMISSIONS_LEVELS = [
+        ("public", "Public"),
+        ("internal", "Internal"),
+        ("private", "Private"),
+    ]
+
+    permissions_level = models.CharField(
+        max_length=8, choices=PERMISSIONS_LEVELS, default="private"
+    )
+
```

Comment:
Idem.

Reply:
The reason I didn't inherit from the base class here is that this one has an extra permission level ("internal") not found in the base class. Is there a better way of doing this?